src/swell/test/suite_tests/suite_tests.py

```diff
@@ -71,12 +135,29 @@ def run_suite(suite: str):
     experiment_dir = testdir / experiment_id
     experiment_dir.mkdir(parents=True, exist_ok=True)
 
+    # Build JEDI for tier 2 tests if existing build is not specified in user yaml
+    if test_tier == TestSuite.TIER2:
+        if ~("jedi_build_method" in test_config
```

Comment:
```suggestion
        if not ("jedi_build_method" in test_config
```

The tilde operator in Python is the invert operation, _not_ (necessarily) a negation. Try this:

```python
if ~True:
  print("yes")
```

Huh? What does `~True` return? And then, [study this](https://stackoverflow.com/questions/8305199/the-tilde-operator-in-python) 😉

Reply:
Oops I forgot about this, thanks